skimage/data/_fetchers.py

```diff
@@ -120,17 +120,12 @@ def create_image_fetcher():
         **retry,
     )
 
-    data_dir = osp.join(str(image_fetcher.abspath), 'data')
+    data_dir = image_fetcher.abspath / 'data'
     return image_fetcher, data_dir
 
 
 image_fetcher, data_dir = create_image_fetcher()
 
-if image_fetcher is None:
-    has_pooch = False
-else:
-    has_pooch = True
```

Comment:
This doesn't seem to be used...

Reply:
It used to be, but it is no longer after your changes.